### PR TITLE
[yoga] Add test to check ceph keys

### PR DESCRIPTION
**Pull request by rodrigogansobarbieri**
For https://review.opendev.org/897549

(cherry picked from commit 912f33e0712034648d393874f14a27cf168bf68b) (cherry picked from commit e242121880ffa1535ad5ed89036a09cc3a6806af) (cherry picked from commit d1a5ab463e3645ddf49818e23ccdea961721c187) (cherry picked from commit e23e14b3607101ade5dec81216d032579c5282ca)

### CODE REVIEW

File: zaza/openstack/charm_tests/nova/tests.py [hunk 1]
```diff
@@ -468,6 +469,63 @@ def test_901_pause_resume(self):
...
+
+        for unit in zaza.model.get_units(
+                self.application_name, model_name=self.model_name):
+
+            # old key
+            check_keyring(self.application_name)
+            # new key
+            check_keyring('nova-compute-ceph-auth-c91ce26f')
+
     def test_930_check_virsh_default_network(self):
```
Comment (dosaboy) [inline zaza/openstack/charm_tests/nova/tests.py:529]: check_keyring is referring to variable "unit" that is from global space. It should be passed into check_keyring() as an argument.

Comment (rodrigogansobarbieri) [reply zaza/openstack/charm_tests/nova/tests.py:529]: omg yes, that slipped by, we will need to fix it in master and bobcat as well

Comment (rodrigogansobarbieri) [reply zaza/openstack/charm_tests/nova/tests.py:529]: actually, I just tested the logic and that coding problem does not affect the result at all

Comment (dosaboy) [reply zaza/openstack/charm_tests/nova/tests.py:529]: yep no worries, can just fix on master if we get time

**Pull request merged**